src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -736,6 +736,16 @@ export class Workbench implements IPartService {
 		// Create Workbench DIV Off-DOM
 		this.workbenchContainer = $('.monaco-workbench-container');
 		this.workbench = $().div({ 'class': 'monaco-workbench ' + (isWindows ? 'windows' : isLinux ? 'linux' : 'mac'), id: Identifiers.WORKBENCH_CONTAINER }).appendTo(this.workbenchContainer);
+
+
+		// Mac specific UI changes
+		if (isMacintosh) {
+			const {serviceCollection} = this.workbenchParams;
+			const windowConfig = this.configurationService.getConfiguration<any>('window');
```

Comment:
Instead of any, you should add the setting to https://github.com/Microsoft/vscode/blob/mac-titlebar-inline/src/vs/workbench/electron-browser/common.ts#L24 and use `IWindowConfiguration` properly.